packages/webapp/src/components/Crop/InGroundTransplant/index.js

```diff
@@ -0,0 +1,195 @@
+import Button from '../../Form/Button';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { Label, Main } from '../../Typography';
+import Form from '../../Form';
+import { useForm } from 'react-hook-form';
+import MultiStepPageTitle from '../../PageTitle/MultiStepPageTitle';
+import RadioGroup from '../../Form/RadioGroup';
+import styles from './styles.module.scss';
+import { ReactComponent as Individual } from '../../../assets/images/plantingMethod/Individual.svg';
+import { ReactComponent as Rows } from '../../../assets/images/plantingMethod/Rows.svg';
+import { ReactComponent as Beds } from '../../../assets/images/plantingMethod/Beds.svg';
+import { ReactComponent as Monocrop } from '../../../assets/images/plantingMethod/Monocrop.svg';
+import { DO_CDN_URL } from '../../../util/constants';
+import ImageModal from '../../Modals/ImageModal';
+import { cloneObject } from '../../../util';
+
+const BROADCAST = 'BROADCAST';
+const CONTAINER = 'CONTAINER';
+const BEDS = 'BEDS';
+const ROWS = 'ROWS';
+const images = {
+  [BROADCAST]: [
+    `${DO_CDN_URL}/planting_method/Broadcast_1.jpg`,
+    `${DO_CDN_URL}/planting_method/Broadcast_2.jpg`,
+    `${DO_CDN_URL}/planting_method/Broadcast_3.jpg`,
+  ],
+  [CONTAINER]: [
+    `${DO_CDN_URL}/planting_method/Individual_1.jpg`,
+    `${DO_CDN_URL}/planting_method/Individual_2.jpg`,
+    `${DO_CDN_URL}/planting_method/Individual_3.jpg`,
+  ],
+  [BEDS]: [
+    `${DO_CDN_URL}/planting_method/Bed_1.jpg`,
+    `${DO_CDN_URL}/planting_method/Bed_2.jpg`,
+    `${DO_CDN_URL}/planting_method/Bed_3.jpg`,
+  ],
+  [ROWS]: [
+    `${DO_CDN_URL}/planting_method/Rows_1.jpg`,
+    `${DO_CDN_URL}/planting_method/Rows_2.jpg`,
+    `${DO_CDN_URL}/planting_method/Rows_3.jpg`,
+  ],
+};
+
+export default function PureInGroundTransplant({
+  useHookFormPersist,
+  persistedFormData,
+  history,
+  match,
```

Comment:
I would move `match` to the parent component and pass in the `variety_id` instead, since that is all the info you're getting from `match`.